### ISSUE TITLE
Issues on Linux

### ISSUE THREAD

**Issue by cjbassi** [davicorreiajr/spotify-now-playing#3]
First off just wanted to say this is a really cool program!

I am running into a few issues on Linux including:
1) The spotify icon isn't visible on a dark background, since the icon is very dark
2) https://github.com/davicorreiajr/spotify-now-playing/blob/d500d86c6adffbfb9b3013313522aabf38d8d305/src/index.js#L127 causes the program to crash on Linux, so the platform should be checked and this command should only run on OSX
3) If I don't have spotify open or spotify is open but a song hasn't started playing since it's been opened, I get an error: `UnhandledPromiseRejectionWarning: Unhandled promise rejection.` so there's a fetch somewhere without a `.catch()`
4) On i3wm, the popup window flashes in the top left corner and then moves to the middle of the screen instead of appearing over the icon.

I'll play around with some of these and make some PR's if I get anywhere.

**Comment by davicorreiajr** [comment]
Thank you so much for your feedback!

As I said in the README, I've done this focused on macOS, first because is my os, secondly because I've basically done in a couple of days and still didn't have time to adapt to other operating systems.

A friend of mine with Linux reported similar things! I'll see if I find some time in the next week, but if you could help with the PR, it would be awesome! :) I'll keep the issue open so we can keep this alive.

Again, thanks!!

**Comment by davicorreiajr** [comment]
hey, I tried to work more on this, and the problem with the icon seems something deeper, quite hard to walk around, as you can see in this issue: https://github.com/electron/electron/issues/6773.

Now, you can run the app locally, but it works very differently from what I'd like to: if you click in the icon, you are going to see the app indicator, and then, if you click in the app-name, the app appears in the left/upper corner (I could't get the "coordinates" of the tray icon on Ubuntu).

In summary, the app works, but really bad. I'm following the issue I mentioned to see if something feasible shows up, and then I can make the app really useful for Ubuntu, because I think now it's quite bad.

I'm gonna close the issue for now, but please let me know if you find a good solution.
